frontend/components/Dashboard/Dashboard.jsx

```diff
@@ -57,6 +62,24 @@ function Sidebar1() {
 }
 
 function Sidebar2() {
+
+ 
+
+
+
+
+    function fetchTotalLinks(){
+        return axios.get(`http://localhost:5000/minify/all`)
```

Comment:
Use the env variable to fetch the API url

Reply:
> Use the env variable to fetch the API url

or i can use "/minify/all" also? another thing how can i access this #NEXT_PUBLIC_API_URL from the .env . it's giving me undefined

Reply:
> Use the env variable to fetch the API url

i have changed this line with axios helper method
![Capture1](https://user-images.githubusercontent.com/82874704/167296281-65df73ec-aa0f-45eb-b1c7-86be968d5f9d.JPG)

Reply:
yeah use this

Reply:
> yeah use this

ok doing the commit